### DISCUSSION TITLE
How to add more height range??

### DISCUSSION THREAD

**Discussion by Alexey8954** [OrcaSlicer/OrcaSlicer#2754] [Q&A]
![Скриншот 15 11 23_23 17 51](https://github.com/SoftFever/OrcaSlicer/assets/150964079/c061771a-a7c1-4ca4-8839-eb5659521b4a)

**Comment by Noisyfox** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/1537155/4e2fc1e0-3dfb-4797-a1a0-48a3d135a231)
Select this first

**Reply by Alexey8954** [reply]
Thanks!!!
